android/expoview/src/main/java/host/exp/exponent/experience/ReactNativeActivity.java

```diff
@@ -159,133 +167,131 @@ public View getRootView() {
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(null);
 
-    mLayout = new FrameLayout(this);
-    setContentView(mLayout);
+    mContainerView = new FrameLayout(this);
+    // TODO (@bbarthec): unify color name R.color.splashBackground
+    mContainerView.setBackgroundColor(ContextCompat.getColor(this, R.color.splashscreen_background));
+    setContentView(mContainerView);
 
-    mContainer = new FrameLayout(this);
-    mLayout.addView(mContainer);
-    mLoadingView = new LoadingView(this);
-    if (!Constants.isStandaloneApp() || Constants.SHOW_LOADING_VIEW_IN_SHELL_APP) {
-      mContainer.setBackgroundColor(ContextCompat.getColor(this, R.color.splashBackground));
-      mLayout.addView(mLoadingView);
+    mReactContainerView = new FrameLayout(this);
+    mContainerView.addView(mReactContainerView);
+
+    // TODO (@bbarthec) if (!Constants.isStandaloneApp()) {
```

Comment:
does this TODO need to be resolved?

Reply:
Yeah, right now, for some unknown reason, I'm cannot create local running standalone build.
Every time I run a standalone app that is built by the local `turtle-cli` (local `turtle-cli` is given modified android shellapp tarball beforehand) I got immediate runtime error:
```
E/AndroidRuntime: FATAL EXCEPTION: create_react_context
    Process: com.swmansion.bbarthec.splashscreen.sdk38, PID: 6945
    java.lang.UnsatisfiedLinkError: couldn't find DSO to load: libhermes.so result: 0
```
@tsapeta promised me that he'll look into it, because I'm out of ideas right now.